views/content/wallet.md

```diff
@@ -0,0 +1,14 @@
+## Ethereum Wallet
+
+The Ethereum Wallet app that you can download allows you to run the full block chain on your own computer. It allows you very easy to be part of the blockchain.
+
+![](/images/wallet-screenshot.png)
+
+With the wallet app you can create new wallets and transfer ether really easily. You can also interact with smart contracts that other have deployed to the Ethereum blockchain.
+
+You don't absolute need the wallet to hold ether. There is different places that also allow you to interact with the Ethereum blockchain such as projects like
+
+- MyEthereumWallet
+- Metamask
+
+Running the wallet app and thereby having the whole blockchain on your computer is called running a "full node". This is crucial to the success of blockchains because your are participating in the network that ensures that there is a single truth.
```

Comment:
Yes, but I feel like it makes sense to talk about since that is what you are doing?